osquery/tables/system/windows/registry.cpp

```diff
@@ -0,0 +1,109 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#include <map>
+#include <sstream>
+#include <stdlib.h>
```

Comment:
Stack your C includes before C++, just pull this `stdlib` (if you think you need it) into it's own section above these C++ headers.